### PR TITLE
[CodeClean] fix svace issue

### PR DESCRIPTION
**Pull request by jaeyun-jung**
Code clean, fix svace issues.
- check max number of tensors and prevent underflow.

### CODE REVIEW

File: gst/nnstreamer/tensor_query/tensor_query_client.c [hunk 1]
```diff
@@ -727,7 +727,8 @@ gst_tensor_query_client_chain (GstPad * pad,
   data_h = g_async_queue_timeout_pop (self->msg_queue,
       self->timeout * G_TIME_SPAN_MILLISECOND);
   if (data_h) {
-    self->requested_num--;
+    if (self->requested_num > 0)
```
Comment (songgot) [inline gst/nnstreamer/tensor_query/tensor_query_client.c:730]: need { } in lines 730 to 769?

Comment (jaeyun-jung) [reply gst/nnstreamer/tensor_query/tensor_query_client.c:730]: yes, pushing buffer to downstream element occurs when a data processed from query-server in the queue.

File: gst/nnstreamer/tensor_query/tensor_query_client.c [hunk 2]
```diff
@@ -727,7 +727,8 @@ gst_tensor_query_client_chain (GstPad * pad,
...
+      self->requested_num--;
     ret = nns_edge_data_get_count (data_h, &num_data);
 
     if (ret == NNS_EDGE_ERROR_NONE && num_data > 0) {
```
Comment (songgot) [inline gst/nnstreamer/tensor_query/tensor_query_client.c:734]: Sorry,
Should code(line 732 ~ 769) be done when ```self->requested_num > 0``` is TRUE ?

Comment (jaeyun-jung) [reply gst/nnstreamer/tensor_query/tensor_query_client.c:734]: Actually unnecessary. The param `self->requested_num` is required for limiting max input to query-server.
query-client handles the response from query-server by using msg-queue.

**Review by songgot — APPROVED** [review]
LGTM

**Comment by SeoHyungjun** [comment]
LGTM

File: ext/nnstreamer/tensor_converter/tensor_converter_python3.cc [hunk 3]
```diff
@@ -173,10 +173,15 @@ PYConverterCore::convert (GstBuffer *in_buf, GstTensorsConfig *config)
 
   if (output) {
     GstTensorInfo *_info;
-    unsigned int num_tensors = PyList_Size (output);
+    unsigned int num_tensors = (unsigned int) PyList_Size (output);
```
Comment (jihochu) [inline ext/nnstreamer/tensor_converter/tensor_converter_python3.cc:176]: Return of `PyList_Size`  coud be negative,  so how about checking error ?

```
unsigned int num_tensors = NNS_TENSOR_SIZE_LIMIT + 1;
Py_ssize_t out_len = PyList_Size (output);
if (out_len >= 0)
   num_tensors = (unsigned int) out_len;
```

Comment (jaeyun-jung) [reply ext/nnstreamer/tensor_converter/tensor_converter_python3.cc]: I changed param type (Py_ssize_t), please review the changes again.

**Review by jihochu — APPROVED** [review]
LGTM

**Comment by myungjoo** [comment]
```
[  503s] [Starting] nnstreamer_filter_python3
[  504s] ==================================================
[  504s]     Test Group nnstreamer_filter_python3 Starts.
[  504s] [PASSED] 1:gst-launch of case 1
[  504s] [PASSED] 1:Compare 1
[  504s] [PASSED] 2:gst-launch of case 2
[  504s] [FAILED][Critical] 2:Golden test comparison
[  504s] [PASSED] 3:gst-launch of case 3
[  504s] [FAILED][Critical] 3:Golden test comparison
[  504s] [IGNORED] 4-1:gst-launch of case 4-1
[  504s] [PASSED] 4-2:Multithreaded python script as a filter (CV2)
[  504s] ==================================================
[  504s] [FAILED] Test Group nnstreamer_filter_python3 has failed cases (3)
```

**Pull request merged**